### DISCUSSION TITLE
Bug: "destination url pathname must be at least 8 characters long"

### DISCUSSION THREAD

**Discussion by jamescridland** [skymethod/op3#6] [General]
Over [here](https://github.com/skymethod/op3/discussions/5#discussion-4421406), @drpancake writes:

> Also I ran into an issue with the ?url=xxx parameter using a wildcard. It often complains that the URL is too short and that makes it difficult to target shows like No Agenda where episode URLs start with only mp3s.nashownotes.com/.

Agree. This looks like a bug.

Compare

`https://op3.dev/api/1/redirect-logs?start=-24h&url=https://op3.dev/e/podnews.net/audio/po*&format=json&token=preview07ce`
... which [works fine](https://op3.dev/api/1/redirect-logs?start=-24h&url=https://op3.dev/e/podnews.net/audio/po*&format=json&token=preview07ce)

...and the longer
`https://op3.dev/api/1/redirect-logs?start=-24h&url=https://op3.dev/e/mp3s.nashownotes.com/PC*&format=json&token=preview07ce`
... which [doesn't](https://op3.dev/api/1/redirect-logs?start=-24h&url=https://op3.dev/e/mp3s.nashownotes.com/PC*&format=json&token=preview07ce)

Error for the second one is `{
  "message": "Bad urlStartsWith: https://op3.dev/e/mp3s.nashownotes.com/PC, destination url pathname must be at least 8 characters long, found /PC"
}`

**Edit:** Having actually *read* the error message now, this isn't a bug, is it - it's working as designed. The Podnews example has a pathname of eight characters: `audio/po` - while the Podcasting 2.0 one has a pathname of only two.

**Comment by johnspurlock-skymethod** [comment]
You got it - right now it's intentional, since the original intention of this search was not to do an expensive broad scan with a non-discriminating filter, but rather to find variants of specific episodes.

I'll take another look at this index again though and see if I can relax it - others are wanting it as well: https://podcastindex.social/web/@js/109064548104573281
